0020-bluetooth-payment-protocol/0020-bluetooth-payment-protocol.md

```diff
@@ -0,0 +1,110 @@
+# Bluetooth Payment Protocol
+
+This document specifies a protocol for communicating and authorizing Interledger Payment Requests over Bluetooth (Low Energy). It is primarily intended for use in in-person retail payment contexts.
+
+This protocol may be implemented by Customer devices such as smartphones or very low-power, offline devices such as [tapFi](https://github.com/joaopedrovbs/tapFi).
+
+Once a Customer becomes active by broadcasting it's payment service over BLE, Merchants can connect
+and request signature for payments over a secure Bluetooth channel. The channel is closed once
+a payment is signed or the Customer cancels the purshase for security or authentication reasons.
+
+## Definitions
+
+- `IPR`: [Interledger Payment Request](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md)
+- `Merchant`: The one requesting a payment
```

Comment:
It will take you to the folder on Github. You'd have to click on the file to view it.

We should consider renaming the files in each folder to README.md if we want them to render automatically. Although this would possibly break historic links.